mount.go

```diff
@@ -122,3 +125,56 @@ func getDeviceForPath(path string) (device, error) {
 
 	return dev, nil
 }
+
+const (
+	procMountsFile = "/proc/mounts"
+
+	fieldsPerLine = 6
+)
+
+const (
+	procDeviceIndex = iota
+	procPathIndex
+	procTypeIndex
+)
+
+func getDevicePathAndFsType(mountPoint string) (devicePath, fsType string, err error) {
+	if mountPoint == "" {
+		err = fmt.Errorf("Mount Point cannot be empty")
+		return
+	}
+
+	var file *os.File
+
+	file, err = os.Open(procMountsFile)
+	if err != nil {
+		return
+	}
+
+	defer file.Close()
+
+	reader := bufio.NewReader(file)
+	for {
```

Comment:
@sboeuf /proc/mounts output can change, but the mount info for this device will not change(It has been mounted by docker and will be unmounted by it as well) Using /proc/mounts is a standard way to get mount info.